### PR TITLE
CSI-1196 fix invalid name

### PR DESCRIPTION
**Pull request by oriyarde**
continuation of https://github.com/IBM/ibm-block-csi-driver/pull/434 and https://github.com/IBM/ibm-block-csi-driver/pull/444

### CODE REVIEW

File: controller/array_action/ds8k_rest_client.py [hunk 1]
```diff
@@ -143,13 +143,13 @@ def get_used_lun_numbers_by_host(self, host_name):
         mappings = self._client.get_mappings_by_host(host_name)
         return [mapping.id for mapping in mappings]
 
-    def create_volume(self, name, capacity_in_bytes, pool_id, tp):
+    def create_volume(self, name, capacity_in_bytes, pool_id, thin_provisioning):
```
Comment (ArbelNathan) [inline controller/array_action/ds8k_rest_client.py:146]: even if tp is `thin_provisioning` I don't think it a good name for the parameter.
tp can be every space efficiency we are supporting - including `ese` that we call thin, so it's not the "cross storage" `thin`..

File: controller/array_action/messages.py [hunk 2]
```diff
@@ -1,70 +1,71 @@
-NoConnectionAvailableException_message = "Currently no connection is available to endpoint: {0}"
+NO_CONNECTION_AVAILABLE_EXCEPTION_MESSAGE = "Currently no connection is available to endpoint: {0}"
```
Comment (ArbelNathan) [inline controller/array_action/messages.py:1]: Do we don't want to remove the upper case?

File: controller/controller_server/csi_controller_server.py [hunk 3]
```diff
@@ -495,7 +495,7 @@ def get_identity_config(self, attribute_name):
         return self.cfg['identity'][attribute_name]
 
     @csi_method(error_response_type=csi_pb2.GetPluginInfoResponse)
-    def GetPluginInfo(self, _, context):
+    def GetPluginInfo(self, _, context):  # pylint: disable=invalid-name
```
Comment (ArbelNathan) [inline controller/controller_server/csi_controller_server.py:498]: why not in the rest of the `spec` funcs?

File: controller/array_action/messages.py (hunk 2 above)
Comment (oriyarde) [reply controller/array_action/messages.py:1]: it's a const and this is the common convention for consts

File: controller/controller_server/csi_controller_server.py (hunk 3 above)
Comment (oriyarde) [reply controller/controller_server/csi_controller_server.py:498]: the linter doesn't complain.
perhaps when these ones are disabled, the class majority becomes `PascalCase`, then the linter gives up or something

File: controller/array_action/messages.py (hunk 2 above)
Comment (ArbelNathan) [reply controller/array_action/messages.py:1]: I meant for the msg

File: controller/controller_server/csi_controller_server.py (hunk 3 above)
Comment (ArbelNathan) [reply controller/controller_server/csi_controller_server.py:498]: weird..

File: controller/array_action/ds8k_rest_client.py (hunk 1 above)
Comment (ArbelNathan) [reply controller/array_action/ds8k_rest_client.py:146]: we would still need to use tp eventually where?

File: controller/array_action/messages.py (hunk 2 above)
Comment (oriyarde) [reply controller/array_action/messages.py:1]: oh, sure, but I think there is enough diff in this PR.
we can align all messages content in a single separate PR

Comment (ArbelNathan) [reply controller/array_action/messages.py:1]: can you open a ticket?

File: controller/array_action/ds8k_rest_client.py (hunk 1 above)
Comment (ArbelNathan) [reply controller/array_action/ds8k_rest_client.py:146]: we are using `tp` there

Comment (oriyarde) [reply controller/array_action/ds8k_rest_client.py:146]: exactly.
so in the function call "flow", the value is initially called "space efficiency", but would eventually have to be called `tp`.
we can either call it "space efficiency" all the way until the last call (shown in the link above), or have this intermediate name to explain why "space efficiency" becomes "tp"

File: controller/array_action/messages.py (hunk 2 above)
Comment (oriyarde) [reply controller/array_action/messages.py:1]: CSI-3899

File: controller/array_action/ds8k_rest_client.py (hunk 1 above)
Comment (ArbelNathan) [reply controller/array_action/ds8k_rest_client.py:146]: but why using `thin_provisioning` and not stay with `tp`?

Comment (oriyarde) [reply controller/array_action/ds8k_rest_client.py:146]: because `tp` is too short, it's not descriptive enough (pylint default minimum length is 3)

Comment (ArbelNathan) [reply controller/array_action/ds8k_rest_client.py:146]: ok, fair enough

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**